auth/token.go

```diff
@@ -83,17 +84,21 @@ func (t stdToken) IssuedAt() time.Time {
 	return t.jwtToken.IssuedAt()
 }
 
-func (t stdToken) Issuer() string {
-	// support ias custom domains
-	iss := t.IasIssuer()
-	if iss == "" {
-		iss = t.jwtToken.Issuer()
+func (t stdToken) CustomIssuer() string {
+	// only return iss if ias_iss does exist
+	_, err := t.GetClaimAsString(claimIasIssuer)
+	if errors.Is(err, ErrClaimNotExists) {
+		return ""
 	}
-	return iss
+	return t.jwtToken.Issuer()
 }
 
-func (t stdToken) IasIssuer() string {
-	v, _ := t.GetClaimAsString(iasIssuer)
+func (t stdToken) Issuer() string {
+	// return standard issuer if ias_iss is not set
+	v, err := t.GetClaimAsString(claimIasIssuer)
+	if errors.Is(err, ErrClaimNotExists) {
```

Comment:
I would prefer a hasClaim method

Reply:
I need the value if it does exist. This way I don't need to get the claim and possibly check the error again.